workspace/__lib__/xod/core/has-error(pulse)/patch.cpp

```diff
@@ -0,0 +1,19 @@
+
+#pragma XOD error_catch enable
+
+struct State {
+};
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
+    if (isInputDirty<input_IN>(ctx) && getError<input_IN>(ctx)) {
+        emitValue<output_OUT>(ctx, true);
+        setTimeout(ctx, 0);
```

Comment:
A comment explaining that we need to manually trigger a reevaluation because pulse errors are cleared at the end of transaction would be nice. There is no way I (and probably anyone else) could explain the reason this line is here in a couple of months.